<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingWindow1</class>
 <widget class="QMainWindow" name="WorkingWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>260</width>
      <height>680</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <widget class="QLineEdit" name="project_field">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>New Project Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="task_field">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>New Task Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_task_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>118</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#add_task_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Add Task</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="task_description">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>240</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="project_description">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>240</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_project_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>118</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#add_project_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Add Project</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans</family>
       <pointsize>18</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <underline>true</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0,0,0);
font-family: &quot;Droid Sans&quot;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="del_user_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#del_user_button {
	color: rgb(255,255,255);
	background-color: rgb(255, 0, 0);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 17px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Delete User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="log_out_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>630</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#log_out_button {
	color: rgb(255,255,255);
	background-color: rgb(255, 0, 0);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 17px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="del_task_button">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>58</y>
      <width>118</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#del_task_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Delete Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_project_button">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>18</y>
      <width>118</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#del_project_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Delete Project</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_task_field">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>58</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Task Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_project_field">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>18</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>431</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="show_project_field">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Project Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_task_field">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>171</y>
       <width>241</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Task Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_project_button">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>10</y>
       <width>118</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#show_project_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Show Project</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_task_button">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>170</y>
       <width>118</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#show_task_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Show Task</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="show_project_description">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>411</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="show_task_description">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>211</y>
       <width>411</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="show_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>411</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="show_all_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#show_all_button {
	color: rgb(255,255,255);
	background-color: rgb(101, 181, 67);
   	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 17px;
    min-width: 6em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Show All Projects and Tasks</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#logo {
border-image: url(:/images/channels4_profile.png);
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
